src/measure_theory/measure/vector_measure.lean

```diff
@@ -1088,6 +1088,15 @@ lemma smul_left {R : Type*} [semiring R] [distrib_mul_action R M] [topological_s
   [has_continuous_smul R M] (r : R) (h : v ⊥ᵥ w) : r • v ⊥ᵥ w :=
 (smul_right r h.symm).symm
 
+lemma neg {M : Type*} [add_comm_group M] [topological_space M] [topological_add_group M]
```

Comment:
Can you call this `neg_left` and add a `neg_right`? (with the obvious proof by symm and neg_left)